packages/components/src/segmented-control/styles.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/core';
```

Comment:
For now this should be imported from `emotion`. We'll change it to `@emotion/core` with the Emotion 11 upgrade.